libsol/util.h

```diff
@@ -7,12 +7,17 @@
         int err = x;         \
         if (err) return err; \
     } while (0)
+
+#ifndef MIN
 #define MIN(a, b) ((a) < (b) ? (a) : (b));
+#endif
```

Comment:
Shouldn't we use the macro already defined in the SDK: https://github.com/LedgerHQ/ledger-secure-sdk/blob/a2be76cb8a7cf66079ad37101c33853a3278d2d3/include/os_math.h#L4 ?

Reply:
`libsol` is also compiled in testU context, without the SDK
I'll overhaul this part later